### DISCUSSION TITLE
Add H264 software encoding as codec option

### DISCUSSION THREAD

**Discussion by 7eventech77** [rustdesk/rustdesk#9904] [General]
I noticed that H264 is only an option as hardware encoding. H264/x264 or whatever library you could use may be easier to encode on a PC without H264 hardware encode compared VP8 or VP9 and have lower bitrate consumption. More (including phones) and older devices also support hardware H264 (Intel 1st gen) decode compared to VP8(Intel 5th gen/VP9 Intel 7th gen) which would easier to decode and have less impact on system resources. Not having H265/H266 is understandable due to the licensing issues with software encoding.

Thank you!

**Comment by rustdesk** [comment]
No, we won't do.
